docs/src/components/Palette/Palette.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  PageBlock,
+  Card,
+  Section,
+  Text,
+  Paragraph
+} from 'seek-style-guide/react';
+import lessToJs from 'less-vars-to-js';
+
+import brand from '!!raw-loader!../../../../theme/palette/brand.less';
+import elements from '!!raw-loader!../../../../theme/palette/elements.less';
+import grays from '!!raw-loader!../../../../theme/palette/grays.less';
+import partners from '!!raw-loader!../../../../theme/palette/partners.less';
+import accessibleVariants from '!!raw-loader!../../../../theme/palette/accessible-variants.less';
+
+import styles from './Palette.less';
+
+const palettes = [
+  { name: 'brand.less', variables: lessToJs(brand) },
+  { name: 'grays.less', variables: lessToJs(grays) },
+  { name: 'elements.less', variables: lessToJs(elements) },
+  { name: 'partners.less', variables: lessToJs(partners) },
+  { name: 'accessible-variants.less', variables: lessToJs(accessibleVariants) }
+];
+
+export default () => (
+  <div>
+    <PageBlock>
+      <Section header>
+        <Text hero>Palette</Text>
+      </Section>
+      <Card transparent style={{ maxWidth: 720 }}>
```

Comment:
Perhaps this style could go in `Palette.less`. Or you could use `AsidedLayout` to wrap the card

Reply:
This inline style was copied from another, existing guide page (Typography). We might need to look at applying a consistent fix across all pages which I'd prefer was done in another PR.